Gemfile

```diff
@@ -74,6 +74,9 @@ gem "pg_search"
 # Hotwire's modest JavaScript framework [https://stimulus.hotwired.dev]
 gem "stimulus-rails"
 
+# User-Resource Permissions and Scoping
+gem "pundit"
+
```

Comment:
This seems like a useful pattern to introduce, but it's the sort of change that I think warrants being documented in an ADR.

If we consider this to be our chosen pattern for handling user authorisation, then it should be adopted wholesale throughout the app.